### PR TITLE
Feature/db schema

### PR DESCRIPTION
**Pull request by keksklauer4**
Worked not only on the db schema but also on several controllers.

### CODE REVIEW

File: src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    @Bean
+    public LinkDiscoverers discoverers() {
+        List<LinkDiscoverer> plugins = new ArrayList<>();
+        plugins.add(new CollectionJsonLinkDiscoverer());
+        return new LinkDiscoverers(SimplePluginRegistry.create(plugins));
+    }
+
+    ApiInfo apiInfo = new ApiInfo( // TODO: work on that
```
Comment (jatsqi) [inline src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java:40]: Please make this a "private static final" attribute in the future

File: src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                "Vaultionizer API",
+                "A safe space for everybody that seeks after privacy.",
+                "1.0.0",
+                "None",
+                new Contact("Julien Meier", "https://vaultionizer.com/", "contact@vaultionizer.com"),
```
Comment (jatsqi) [inline src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java:45]: I think "Team Vaultionizer" or "Vaultionizer" would be better here

**Review by jatsqi — CHANGES_REQUESTED** [review]

File: src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java (hunk 1 above)
Comment (keksklauer4) [reply src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java:40]: I will change that in another feature branch. I was just working on fully enabling Swagger and correctly documenting the current endpoints ;)

File: src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java (hunk 2 above)
Comment (keksklauer4) [reply src/main/java/com/vaultionizer/vaultserver/config/SwaggerConfig.java:45]: I will do that in another branch.

**Review by jatsqi — APPROVED** [review]

**Pull request merged**